javascript/javascript1/week1/EzNamey.js

```diff
@@ -0,0 +1,6 @@
+//startup name generator
+const firstWords = ['Easy','Awesome','Corporate','Company','Startup','Software','Information Technology','Cognizant','TCS','Infosys'];
+const secondWords = ['Red','blue','yellow', 'green', 'Violet','Circle','Square','Triangle','Rectangle', 'Oval'];
+const randomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
Maybe you could try to simplify with this ? What do you think
Also it is  a good practice to not use constants when you are trying to access indexes in an array
```suggestion
const randomNumber = Math.random() *firstWords.length ;
```

Reply:
Yes, Made the changes